{{cookiecutter.project_slug}}/htdocs/static/scss/_base.scss

```diff
@@ -0,0 +1,108 @@
+body {
+  height: 100%;
+  font-family: $base-font, sans-serif;
```

Comment:
Let base font decide the font type. It might be a serif one. i.e. just have `$base-font` that variable can have the bit after